content/docs/command-reference/plots/index.md

```diff
@@ -125,8 +125,8 @@ All metrics files given to `dvc plots show` and `dvc plots diff` as input are
 combined together into a single data array for injection into a template file.
 There are two important fields that DVC adds to the plot data:
 
-- `index` - self-incrementing, zero-based counter for the data rows/values. In
-  many cases it corresponds to a machine learning training epoch or step number.
+- `index` - zero-based counter for the data rows/values. In many cases it
```

Comment:
In my experience "self-incrementing" (more commonly "auto-increment" means something else: when you insert records (e.g. to a DB table) and the index field gets an automatic value. Numbers may be skipped if records are deleted too.

In this case all the data is already there and we're just assigning a linear integer scale to the automatic index field.

Reply:
kk